Sources/PointFree/Blog/BlogPostIndex.swift

```diff
@@ -0,0 +1,65 @@
+import Css
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import Prelude
+import Styleguide
+import Tuple
+
+let blogIndexMiddleware: (Conn<StatusLineOpen, Tuple3<Database.User?, SubscriberState, Route?>>) -> IO<Conn<ResponseEnded, Data>> =
+  writeStatus(.ok)
+    >-> map(lower)
+    >>> respond(
+      view: blogIndexView,
+      layoutData: { currentUser, subscriberState, currentRoute in
+        SimplePageLayoutData(
+          currentRoute: currentRoute,
+          currentSubscriberState: subscriberState,
+          currentUser: currentUser,
+          data: (currentUser, subscriberState),
+          // TODO
+          description: "Point-Free is a video series exploring functional programming and Swift.",
+          extraStyles: markdownBlockStyles,
+          // TODO
+          image: "https://d3rccdn33rt8ze.cloudfront.net/social-assets/twitter-card-large.png",
+          navStyle: .mountains(.blog),
+          openGraphType: .website,
+          // TODO
+          title: "Point-Free: A video series on functional programming and the Swift programming language.",
+          twitterCard: .summaryLargeImage
+        )
+    }
+)
+
+private let blogIndexView = View<(Database.User?, SubscriberState)> { currentUser, subscriberState in
```

Comment:
not currently using this data in this view, but will soon to have subscriber callouts